code/python/setup.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python
```

Comment:
Notes on the readme suggested a few things we may want to specify here such as whether it only runs on Linux and known compatible Python version(s).